src/index/CompressedRelation.h

```diff
@@ -259,7 +259,7 @@ class CompressedRelationReader {
     size_t numBlocksRead_ = 0;
     size_t numBlocksAll_ = 0;
     size_t numElementsRead_ = 0;
-    size_t blockingTimeMs_ = 0;
+    std::chrono::milliseconds blockingTime_ = std::chrono::milliseconds::zero();
```

Comment:
0 ms.

Reply:
Can't do without adding it to the global namespace

Reply:
This problem probably will also be used by C++20 modules I guess
(Cluttering the global namespace not only with macros, but also with these very useful literal namespaces